google/cloud/documentai_toolbox/wrappers/entity.py

```diff
@@ -57,28 +57,33 @@ class Entity:
     page_offset: dataclasses.InitVar[Optional[int]] = 0
 
     type_: str = dataclasses.field(init=False)
-    mention_text: str = dataclasses.field(init=False, default="")
-    normalized_text: str = dataclasses.field(init=False, default="")
+    mention_text: Optional[str] = dataclasses.field(init=False, default=None)
+    normalized_text: Optional[str] = dataclasses.field(init=False, default=None)
 
-    start_page: int = dataclasses.field(init=False)
-    # Only Populated for Splitter/Classifier Output
-    end_page: int = dataclasses.field(init=False)
+    # Not populated in Classifier output
+    start_page: Optional[int] = dataclasses.field(init=False, default=None)
+    # Only Populated for Splitter Output
+    end_page: Optional[int] = dataclasses.field(init=False, default=None)
 
     _image: Optional[Image.Image] = dataclasses.field(init=False, default=None)
 
     def __post_init__(self, page_offset: int) -> None:
         self.type_ = self.documentai_object.type_
-        self.mention_text = self.documentai_object.mention_text
+
+        if self.documentai_object.mention_text:
```

Comment:
Is the point here that if mention_text is the empty string, we want to keep self.mention_text as `None`, so that we will be able to detect this by checking `self.mention_text is None` downstream?

Reply:
Yes, that's why. Depending on how the output JSON is formatted, the `mention_text` field can be either `None` or the empty string and we want to keep it as `None` for an empty input.